src/components/NoteList.js

```diff
@@ -52,7 +43,14 @@ class NoteList extends Component {
           />
         </form>
         <ul>
-          {this.props.notes.map(note => <NoteItem key={note.id} {...note} />)}
+          {this.props.notes.map(note => (
+            <NoteItem
+              key={note.id}
+              toggleNote={this.props.toggleNote}
+              deleteNote={this.props.deleteNote}
```

Comment:
@laurelherting This is where i'm passing `deleteNote` down to the individual NoteItems in the file `Note.js`